mkdocs.yml

```diff
@@ -35,17 +35,18 @@ markdown_extensions:
   - pymdownx.highlight:
       linenums: true
   - pymdownx.superfences
+  - admonition
+  - pymdownx.tabbed
 
 nav:
-  - Home: 'index.md'
-  - 'User guide':
-      - 'ABCs of MongooseIM': 'user-guide/ABCs-of-MongooseIM.md'
+  - 'Home': 'index.md'
+  - 'Getting Started':
+      - 'Installation': 'getting-started/Installation.md'
+      - 'Quick Setup': 'getting-started/Quick-setup.md'
+  - 'User Guide':
+      - 'Features': 'user-guide/Features.md'
+      - 'Supported Standards': 'user-guide/Supported-standards.md'
       - 'High-level Architecture': 'user-guide/MongooseIM-High-level-Architecture.md'
-      - 'Features and supported standards': 'user-guide/Features-and-supported-standards.md'
-      - 'Getting started': 'user-guide/Getting-started.md'
-      - 'Release/Installation configuration': 'user-guide/release_config.md'
-      - 'Bootstrap scripts': 'user-guide/Bootstrap-Scripts.md'
-      - 'Client certificate authentication': 'user-guide/client-certificate.md'
   - Tutorials:
       - 'How to Build MongooseIM from source code': 'user-guide/How-to-build.md'
```

Comment:
I'd reorganise the file locations as well, now we'd have for example the `How-to-build` page in the Tutorials section but in the user-guide directory, which is confusing, also because their filepath is what is later shown in the URL. I think the sections in the site and the directories should match.

Reply:
I agree, I was thinking there might have been a reason to have tutorials in the `user-guide` directory, so I left it as it was. But I'm glad you see it this way, will be reorganised.

Reply:
I've also renamed "advanced-configuration" to just "configuration"